easybuild/easyconfigs/i/imkl/imkl-2020.4.304-gompi-2020b.eb

```diff
@@ -11,7 +11,7 @@ description = """Intel Math Kernel Library is a library of highly optimized,
 
 toolchain = {'name': 'gompi', 'version': '2020b'}
 
-source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/16917/']
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/tec/16533/']
```

Comment:
This should not be changed? The source URL is correct, the `16533` one is for `imkl` v2020.1.217...